src/main/org/apache/tools/ant/taskdefs/MacroInstance.java

```diff
@@ -221,8 +221,6 @@ private String macroSubs(String s, Map<String, String> macroMapping) {
             }
         }
         switch (state) {
-            case STATE_NORMAL:
-                break;
```

Comment:
To me the explicit case makes it easier to see what happens than having to look up the default case.

Reply:
It is explicit now, just as in above case, only merged with default

Reply:
doesn't make me like it much more